articles/azure/azs-how-deploy-asdk.md

```diff
@@ -152,90 +260,57 @@ At C:\CloudDeployment\Setup\DeploySingleNode.ps1:676 char:5
  + FullyQualifiedErrorId : Unspecified error,CloudEngine.Cmdlets.InvokeCmdlet
 ```
 
-Verification of Physical kit showed that Virtualisation is in fact enabled but Hyper-V is not starting on boot.
+Verification of Physical kit showed that virtualisation is in fact enabled but Hyper-V is not starting on boot.
 
-To verify run you can run systeminfo and it will show whether BIOS is set correctly etc... In our case it was set correctly. Only Boot settings were not.
+To verify, you can run systeminfo and it will show whether BIOS is set correctly, and so on. In our case it was set correctly, only the boot settings were not.
 
-To fix it run from elevated command/powershell prompt:
+To fix it, run the below from an elevated command/powershell prompt:
```

Comment:
just have PowerShell prompt

Reply:
Changed to just command prompt as I think it's more appropriate for the command we're running.